src/profile_service/index.js

```diff
@@ -15,6 +16,8 @@ const ActivityDetails = function main(parsedResponse, decryptedProfile) {
     acc[propName] = current[propName];
     return acc;
   }, {});
+  const age = new Age(this.profile);
+  this.profile.isAgeVerified = age.isVerified();
```

Comment:
Can't we add a conditional here `if (age.isVerified() !== null)`? It should not add the property if age verification does not exist. Adding this null property breaks the tests as well.

Reply:
I have double checked I can see it's not adding the attribute to the profile if the value is not selected in the Dashboard - I could add an if there.

Reply:
What do you mean by consistent? The other attributes are not being sent if they are null.